aeg.sql

```diff
@@ -119,6 +120,20 @@ CREATE TABLE `farm_economics` (
   `created_at` timestamp
 );
 
+
+CREATE TABLE `weather_stations` (
+	`id` int PRIMARY KEY AUTO_INCREMENT,
+	`user_id` int,
+	`longitude` varchar(255),
+	`latitude` varchar(255),
+    constraint fk_user_id_in_weather_stations
+    foreign key(user_id)
```

Comment:
Let's name user_id to fk_weather_stations_user_id or fk_user_id_in_weather_stations